autonomous/autonomous.py

```diff
@@ -148,8 +148,10 @@ def move(self, tm: float) -> None:
             abs(translation_error) < self.ALLOWED_TRANS_ERROR
             and abs(rotation_error) < self.ALLOWED_ROT_ERROR
         )
+        is_stopped = abs(self.chassis.translation_velocity.norm()) < 0.2
```

Comment:
Isn't the norm of a (real) vector by definition a non-negative real number?
```suggestion
        is_stopped = self.chassis.translation_velocity.norm() < 0.2
```